### ISSUE TITLE
README is a bit confusing about trace-level order

### ISSUE THREAD

**Issue by teetak01** [ARMmbed/mbed-trace#56]
Info level is mentioned in documentation as step above cmd_printf, even through it should be the second highest.

**Comment by ciarmcom** [comment]
ARM Internal Ref: IOTSYST-1874

**Comment by jupe** [comment]
it's actually kind of correct, maybe little confusion way documented. When tracing with level `TRACE_LEVEL_CMD` it behaviour little differently when it is printed out, it uses cmd_printf function pointer which can be define by `mbed_trace_cmdprint_function_set()` -function. This level should not be used in normal case, there was special use case for it in first place.

**Comment by jupe** [comment]
Is there something else what you would like to know ?

**Comment by teetak01** [comment]
I mean the order they are mentioned is wrong:

```
    debug
    warning
    error
    info
    cmdline (special behavior, should not be used)
```
Should be:
```
    debug
    info
    warning
    error
    cmdline (special behavior, should not be used)
```

**Comment by jupe** [comment]
aa, sorry, I little misunderstand :). Could you create fix PR for doc ? ;)

**Comment by tommikas** [comment]
Fixed in #57.
